src/databricks/labs/ucx/install.py

```diff
@@ -812,6 +812,26 @@ def latest_job_status(self) -> list[dict]:
                 continue
         return latest_status
 
+    def repair_run(self, workflow):
+        try:
+            job_id = self._state.jobs.get(workflow)
+            if not job_id:
+                logger.warning(f"{workflow} job does not exists hence skipping Repair Run")
+            else:
+                job_runs = list(self._ws.jobs.list_runs(job_id=job_id, limit=1))
+                if not job_runs:
+                    logger.warning(f"{workflow} job is not initialized yet. Can't trigger repair run now")
+                else:
```

Comment:
You don't need "else" here, just de-dent the next code block

Reply:
Changed according